packages/contracts-bedrock/src/cannon/MIPS2.sol

```diff
@@ -52,7 +52,7 @@ contract MIPS2 is ISemver {
 
     /// @notice The semantic version of the MIPS2 contract.
     /// @custom:semver 0.0.2-beta
-    string public constant version = "0.0.2-beta";
+    string public constant version = "1.0.0-beta.3";
```

Comment:
fyi @Inphi - bumped this according to [this doc](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/VERSIONING.md#individual-contract-versioning):
> New contracts start at 1.0.0-beta.1, increment the -beta.n counter during development, and become 1.0.0 when they are ready for production.